preflight-checks/dcgm-diag/dcgm_diag/diag.py

```
@@ -78,6 +79,13 @@ def _run_diagnostic(self, level: int) -> list[DiagResult]:
 
         log.info(f"Running DCGM diagnostic level={level} gpus={gpu_indices}")
 
+        # Stop any zombie diagnostic left by a previous crashed container on this
+        # node.  Without this, RunDiagnostic fails with "already running".
+        try:
+            dcgm_agent.dcgmStopDiagnostic(self._handle.handle)
+        except Exception:
+            pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)